### PR TITLE
Create Identity method added with Vector functionality

### PR DESCRIPTION
**Pull request by sKudryashov**
test_create_did_doc_vec added because Vec_uint8_t *did_doc_rsp type is needed on the receiver side

### CODE REVIEW

File: ffi/src/did.rs [hunk 1]
```diff
@@ -56,3 +56,33 @@ pub fn create_identity(
     debug!("create_identity None result");
     None
 }
+
+#[ffi_export]
+pub fn create_identity_vec(
```
Comment (LuisOsta) [inline ffi/src/did.rs:61]: Why is this needed btw?

**Review by LuisOsta — APPROVED** [review]

File: ffi/src/did.rs [hunk 2]
```diff
@@ -56,3 +56,33 @@ pub fn create_identity(
...
+        did_method=?did_method,
+        "creating bytes vector did_doc representation");
+    if let Some(did_doc) = t_identity {
+        info!("did_doc received");
+        let did_doc_vector = serde_json::to_vec(&did_doc.backend);
+
+        if did_doc_vector.is_ok() {
+            info!("did_doc converted to vector");
+            let did_doc_c_vec: safer_ffi::vec::Vec<u8> =
+                did_doc_vector.expect("unrapping did doc_vector").into();
```
Comment (fkim7) [inline ffi/src/did.rs:77]: [nit] spelling- unwrapping
